src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,535 @@
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Open menu"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__body">
+                <img
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                    alt="Image"
+                >
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-overflow-vertical-16"/>
+                    </svg>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Image with Preview</h3>
+
+    <p>
+        If "Preview" action button is enabled, it will show on the bottom left with the file extension.
+    </p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__body">
+                        <img
+                            src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                            alt="Image"
+                        >
+                    </div>
+                    <div class="file-preview-card__menu-action">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Open menu"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                    <div class="file-preview-card__preview-action">
+                        <button class="icon-btn" type="button">
```

Comment:
So, we are using an Icon Button here, but this doesn't appear to be an icon button (it contains text). I see a similar thing in the video preview also containing an icon + text and overrides of .icon-btn class. I'm not sure that using icon-btn here is the right move. @ArtBlue @agliga

Reply:
Originally I used ebay-icon-button was because of the hover styles that matched the design and the component does give the capability of providing renderBody